ui/components/members/AllocateModal.js

```diff
@@ -61,9 +73,23 @@ const AllocateModal = ({ member, event, handleClose }) => {
             autoFocus
             className="w-36 mx-auto mb-2"
           />
-          <p className="text-center mb-4">
-            = {thousandSeparator(total / 100)} {event.currency} total
+          <p className="text-center mb-4 text-sm text-gray-800">
+            {type === "Add" ? (
+              <>
+                Adding {thousandSeparator(amount / 100)} {event.currency} to{" "}
```

Comment:
Maybe rename `amount` to cents or something? Idk not super needed but I started mixing up all the different variable names when I got to this part :p

Reply:
All amounts are in cents so renaming to that might just make it more confusing 🤔

Reply:
Yeah good point. Maybe just `amount` that's in full units